docs/start/index.rst

```diff
@@ -191,24 +191,46 @@ are two widely  used options:
  - **Jupyter**, a browser based notebook
  - **Spyder**, an integrated development environment
 
-Both can be installed using conda or pip.
+Both can be installed using ``conda`` or ``pip`` in the created conda environment
+for QCoDeS. Then installation can be simply done by activating the QCoDeS environment
+in the terminal and running the following:
 
-To start either of them you can use the shortcuts in the start menu under
-*Anaconda3* with a trailing *(qcodes)*.
+.. code:: bash
+
+    pip install spyder
+    pip install jupyter
+
+If you installed QCoDeS using conda-forge, you may want to install above using
+``conda install`` rather than ``pip install``. After installation, they could be easily
+called in your QCoDeS-activated conda terminal.
 
-For other options you can launch a terminal either via the *Anaconda Navigator*
-by selecting *qcodes* in the *Environments tab* and left-clicking on the *play*
-button or by entering
+For running spyder:
 
 .. code:: bash
 
-    conda activate qcodes
+    spyder
+
+or for jupyter notebook:
+
+.. code:: bash
+
+    jupyter notebook
+
+or jupyter lab:
```

Comment:
Nitpick I don't think jupyter lab works unless you install jupyterlab

Reply:
Let me double check that.

Reply:
You are right. Mentioned the installation in the last commit.
Also added a line in the Anaconda section to say to be carefull about using it.